components/tidb_query_datatype/src/codec/mysql/json/binary.rs

```diff
@@ -17,7 +17,7 @@ impl<'a> JsonRef<'a> {
 
     /// Return the `i`th key in current Object json
     ///
-    /// See `arrayGetElem()` in TiDB `json/binary.go`
+    /// See `objectGetKey()` in TiDB `json/binary.go`
```

Comment:
Do you mean  `objectGetKey()` in TiDB `json_binary.go？

Reply:
I changed json/binary.go to types/json_binary.go